### PR TITLE
Tickets/dm 34471

### PR DESCRIPTION
**Pull request by patrickingraham**

### CODE REVIEW

File: Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst [hunk 1]
```diff
@@ -57,17 +57,26 @@ Setup and Start DDS Daemon
...
   export OSPL_URI=$(python -c "from lsst.ts import ddsconfig; print( (ddsconfig.get_config_dir() / 'ospl-shmem.xml').as_uri())")
 
+ 
+.. important:: 
+
+   If running on the TTS, substitute the following for the OSPL_HOME definition:
+
+   .. prompt:: bash
+
+      export OSPL_HOME=/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux
```
Comment (tribeiro) [inline Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst:78]: This is not needed on TTS as well..

File: Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst [hunk 2]
```diff
@@ -57,17 +57,26 @@ Setup and Start DDS Daemon
...
-	This step needs to be performed every time the user nublado instance is restarted, or if the user shuts down the daemon.
+	This step needs to be performed every time the user Nublado instance is restarted, or if the user shuts down the daemon.
 
 In the terminal enter the following sequence of commands to setup your environment;
 
 .. prompt:: bash
 
   source ${LOADSTACK}
-  export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/
+  export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux
```
Comment (tribeiro) [inline Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst:67]: You can remove this line as well

File: Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst [hunk 3]
```diff
@@ -57,17 +57,17 @@ Setup and Start DDS Daemon
...
 
   source ${LOADSTACK}
-  export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/
   source $OSPL_HOME/release.com
   export OSPL_URI=$(python -c "from lsst.ts import ddsconfig; print( (ddsconfig.get_config_dir() / 'ospl-shmem.xml').as_uri())")
 
+
 Before starting the daemon, make sure to check if it is already running or not;
 
 .. code-block:: shell-session
```
Comment (tribeiro) [inline Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst:73]: There is one more 

```
export OSPL_HOME=/scratch/opt/OpenSpliceDDS/V6.10.4/HDE/x86_64.linux/
```

in line 122 that can be removed as well.. (I can't comment on it.. :( )

Comment (patrickingraham) [reply Observing-Interface-Setup/setup-dds-daemon-in-nublado.rst:73]: I think the only mention left of OSPL_HOME being defined (in the repos I have checked out) is in ts_cycle_build ... but presumably that has to be there :wink:

**Comment by patrickingraham** [comment]
Build is failing due to the gemini site being down...

**Review by tribeiro — APPROVED** [review]
Looks good!

**Comment by tribeiro** [comment]
Maybe add gemini site to https://github.com/lsst-ts/observatory-ops-docs/blob/3e844fb08adfffeed85938ce33b1954fb76c74c5/conf.py#L119 ?

**Pull request merged**